Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
```

Comment:
@othiym23 I have a problem with the logic here -- we're testing the bundled npm works as it should with io.js but we should also be testing that it's bundled properly and ready to release yet we have an `npm install` inside the bundled directory which could potentially be fixing any bundled dependency problems that may exist. Is there a better way to approach this that would leave the deps/npm directory untouched?